modules/router-performance-optimizations.adoc

```diff
@@ -1,70 +1,10 @@
-router's deployment configuration file.
-
-[NOTE]
-====
-A connection includes the front end and internal back end. This counts as two
-connections. Be sure to set `ROUTER_MAX_CONNECTIONS` to double than the number
-of connections you intend to create.
-====
-
-*CPU and interrupt affinity*
-
-In {product-title}, the HAProxy router runs as a single process. The
-{product-title} HAProxy router typically performs better on a system with fewer
-but high frequency cores, rather than on an symmetric multiprocessing (SMP)
-system with a high number of lower frequency cores.
-
-Pinning the HAProxy process to one CPU core and the network interrupts to
-another CPU core tends to increase network performance. Having processes and
-interrupts on the same non-uniform memory access (NUMA) node helps avoid memory
-accesses by ensuring a shared L3 cache. However, this level of control is
-generally not possible on a public cloud environment.
-
-CPU pinning is performed either by `taskset` or by using HAProxy's `cpu-map`
-parameter. This directive takes two arguments: the process ID and the CPU core
-ID. For example, to pin HAProxy process `1` onto CPU core `0`, add the following
-line to the global section of HAProxy's configuration file:
-
-----
-    cpu-map 1 0
-----
-
-*Increasing the number of threads*
-
-The HAProxy router comes with support for multithreading in {product-title}. On
-a multiple CPU core system, increasing the number of threads can help the
-performance, especially when terminating SSL on the router.
-
-*Impacts of buffer increases*
-
-The {product-title} HAProxy router request buffer configuration limits the size
-of headers in incoming requests and responses from applications. The HAProxy
-parameter `tune.bufsize` can be increased to allow processing of larger headers
-and to allow applications with very large cookies to work, such as those
-accepted by load balancers provided by many public cloud providers. However,
-this affects the total memory use, especially when large numbers of connections
-are open. With very large numbers of open connections, the memory usage will be
-nearly proportionate to the increase of this tunable parameter.
-
-*Optimizations for HAProxy reloads*
-
-Long-lasting connections, such as WebSocket connections, combined with
-long client/server HAProxy timeouts and short HAProxy
-reload intervals, can cause instantiation of many HAProxy processes.
-These processes must handle old connections, which were started
-before the HAProxy configuration reload. A large number of these processes is
-undesirable, as it will exert unnecessary load on the system and can
-lead to issues, such as out of memory conditions.
-
-Router environment variables affecting this
-behavior are `ROUTER_DEFAULT_TUNNEL_TIMEOUT`, `ROUTER_DEFAULT_CLIENT_TIMEOUT`,
-`ROUTER_DEFAULT_SERVER_TIMEOUT`, and `RELOAD_INTERVAL` in particular.
+You can modify the deployment, but if the Operator is enabled, the configuration
```

Comment:
@jmencak What is the Operator's official name? Trying to get a cluster up to check myself, but it keeps failing.

Reply:
The ingress operator?

https://github.com/openshift/cluster-ingress-operator 

```
$ oc get pods -n openshift-ingress-operator
NAME                                READY   STATUS    RESTARTS   AGE
ingress-operator-7c5fd8bbfb-x74dm   1/1     Running   0          7h39m
```